build_tools/circle/build_doc.sh

```diff
@@ -145,8 +145,8 @@ fi
 
 MINICONDA_PATH=$HOME/miniconda
 # Install dependencies with miniconda
-wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
-   -O miniconda.sh
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh \
+    -O miniconda.sh
```

Comment:
O_o do we want to switch to mamba? why not miniforge?

Reply:
It was for debugging reasons first. Then, I assume that `mamba` will be faster than `conda` to resolve the dependencies required here.